src/features/post/post-form/model/usePostForm.ts

```diff
@@ -240,6 +242,16 @@ export const usePostForm = ({ mode, boardId, postId }: Props) => {
               console.error('일정 생성 실패:', err);
             }
           }
+        } else {
+          if (postSchedule?.scheduleId) {
+            await deleteScheduleMutate({
+              postId: numericPostId!,
+              scheduleId: postSchedule.scheduleId,
```

Comment:
```suggestion
            await deleteScheduleMutate({
              postId: targetPostId!,
              scheduleId: postSchedule.scheduleId,
```
똑같은거긴 한데 다른 훅 인자랑 통일하게 요렇게 가는건 어떨까요?